public/app/features/search/page/components/SearchResultsTable.tsx

```diff
@@ -67,8 +78,35 @@ export const SearchResultsTable = React.memo(
       if (listRef.current) {
         listRef.current.scrollTo(0);
       }
+      highlightIndexRef.current = -1;
     }, [memoizedData]);
 
+    const keyEvent = useObservable(keyboardEvents ?? of());
+    useEffect(() => {
+      switch (keyEvent?.code) {
+        case 'ArrowDown': {
+          highlightIndexRef.current += 1;
+          setHighlightIndex(highlightIndexRef.current);
+          break;
+        }
+        case 'ArrowUp':
+          highlightIndexRef.current = Math.max(0, highlightIndexRef.current - 1);
+          setHighlightIndex(highlightIndexRef.current);
+          break;
+        case 'Enter':
+          if (highlightIndexRef.current >= 0 && urlsRef.current) {
+            const url = urlsRef.current.values?.get(highlightIndexRef.current) as string;
+            if (url) {
+              if (url.startsWith(config.appSubUrl)) {
```

Comment:
@torkelo -- locationService.push not working when running from app url 😬, I'm not sure what the best fix is for this